grid/pdf-export-server/Server/Controllers/ExportPdfController.cs

```diff
@@ -0,0 +1,70 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using PdfExport;
+using ServerPdfExport.Server.Services;
+using ServerPdfExport.Shared;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Telerik.DataSource;
+using Telerik.DataSource.Extensions;
+
+namespace ServerPdfExport.Server.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class ExportPdfController : ControllerBase
+    {
+        private readonly ILogger<WeatherForecastController> logger;
+
+        private WeatherForecastDataService MyDataService { get; set; }
+
+        public ExportPdfController(ILogger<WeatherForecastController> logger, WeatherForecastDataService dataService)
+        {
+            this.logger = logger;
+            MyDataService = dataService;
+        }
+
+        [HttpPost]
+        [Route("Direct")]
```

Comment:
The overall idea implies that a better name from this action is `FromPdf`, "Direct" is not descriptive enough. Can also be changed to something containing a verb.